_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title:  "Book Review: The Intelligent Investor"
+date:   2019-02-24 22:40:05
+categories: jekyll update
+---
+
+### Disclaimer and Basics on Retirement
+
+I am simply an enthusiastic lay person
+who happens to know some math, a fair bit of personal finance,
+and likes reading books on the topic of investing. Please do your own research
+and don't ask me for specific advice about any companies,
+markets, etc.
+
+**I will not help you with individual financial stuff. Please do not ask.** Consult with a fiduciary advisor with any specific question you have and make sure your 401k company is not screwing you over. *Hint*: They are probably screwing you over, watch [The Retirement Gamble](https://www.pbs.org/video/frontline-retirement-gamble/).
+
+
+### The Gist
+
+_The Intelligent Investor_ and _Security Analysis_ is the only investment books I ever needed to read, all the others were further elaboration on Benjamin Graham's points. I decided to do this book review because the principles to preserve wealth, reducing the amount of false positives, and truly invest for the long run (the only investment that exists) are timeless techniques. For the purposes of brevity I will focus primarily on what Benjamin Graham calls a Defensive Investor instead of the Enterprising Investor.
```

Comment:
` (the only investment that exists)`

Suggestion: Another means of communicating these asides could be footnotes (instead of mere parentheticals).
 
In this instance, I think this would fit better in a footnote since you already have a lot of information in this sentence. Plus, you can add a link in the footnote justifying this claim -- which it seems like you have on hand, given the confidence of this statement. 
Further, it would read *stronger* if the sentence flowed to "are timeless techniques" sooner.